src/index.js

```diff
@@ -13,12 +16,21 @@ new window.p5(p5Inst => {
 
   p5Inst.preload = nativeAPI.preload.bind(nativeAPI);
   p5Inst.setup = () => {
-    nativeAPI.setMetadata_('hammer', {'bpm': '133', 'delay': '2.32'});
+    nativeAPI.setMetadata_('macklemore90', macklemore90);
+    nativeAPI.setMetadata_('hammer', hammer);
+    nativeAPI.setMetadata_('peas', peas);
+    nativeAPI.addCues({
+      seconds: [],
+      measures: [],
+      peaks: []
+    });
     nativeAPI.setup();
 
     // Sample user code:
     nativeAPI.setBackgroundEffect('disco');
     nativeAPI.makeNewDanceSprite("CAT", null, {x: 200, y: 200});
+
+    nativeAPI.songStartTime_ = new Date();
```

Comment:
Why do we have to reach in to this private API?

Reply:
this is to simulate a song starting (so we can see the peaks and the effects start doing something). We don't really _have_ to, but it sure is nice. I'll add a comment...

Reply:
Better to call `nativeAPI.play()`?

Reply:
yes, that works, since you already stubbed out `playSound()`